Index: graphics/mesa-dri/Makefile
===================================================================
--- graphics/mesa-dri/Makefile
+++ graphics/mesa-dri/Makefile
@@ -3,7 +3,6 @@
 
 PORTNAME=	mesa-dri
 PORTVERSION=	${MESAVERSION}
-PORTREVISION=	9
 CATEGORIES=	graphics
 
 COMMENT=	OpenGL hardware acceleration drivers for DRI2+
Index: graphics/mesa-dri/Makefile.common
===================================================================
--- graphics/mesa-dri/Makefile.common
+++ graphics/mesa-dri/Makefile.common
@@ -14,7 +14,7 @@
 MESAVERSION=	${MESABASEVERSION}${MESASUBVERSION:C/^(.)/.\1/}
 MESADISTVERSION=${MESABASEVERSION}${MESASUBVERSION:C/^(.)/-\1/}
 
-MESABASEVERSION=	18.3.2
+MESABASEVERSION=	18.3.6
 # if there is a subversion, don't include the '-' between 7.11-rc2.
 MESASUBVERSION=
 
Index: graphics/mesa-dri/distinfo
===================================================================
--- graphics/mesa-dri/distinfo
+++ graphics/mesa-dri/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1547724382
-SHA256 (mesa-18.3.2.tar.xz) = f7ce7181c07b6d8e0132da879af1729523a6c8aa87f79a9d59dfd064024cfb35
-SIZE (mesa-18.3.2.tar.xz) = 11868440
+TIMESTAMP = 1554461603
+SHA256 (mesa-18.3.6.tar.xz) = aaf17638dcf5a90b93b6389e152fdc9ef147768b09598f24d2c5cf482fcfc705
+SIZE (mesa-18.3.6.tar.xz) = 11882744
Index: graphics/mesa-libs/Makefile
===================================================================
--- graphics/mesa-libs/Makefile
+++ graphics/mesa-libs/Makefile
@@ -3,7 +3,6 @@
 
 PORTNAME=	mesa-libs
 PORTVERSION=	${MESAVERSION}
-PORTREVISION=	3
 CATEGORIES=	graphics
 
 COMMENT=	OpenGL libraries that support GLX and EGL clients
Index: lang/clover/Makefile
===================================================================
--- lang/clover/Makefile
+++ lang/clover/Makefile
@@ -2,7 +2,6 @@
 
 PORTNAME=	clover
 PORTVERSION=	${MESAVERSION}
-PORTREVISION=	5
 CATEGORIES=	lang
 
 COMMENT=	Mesa "Clover" OpenCL library
